main.py

```diff
@@ -0,0 +1,267 @@
+"""main.py
+
+objects: screen, player, follower
+
+TODO:
+-walls
+-fix jumping with multiple grounds
+DONE:
+- simple objects for player sprite
+- train constant speed
+- player slows down
+- lose on collision
+- flat level
+"""
+
+import pygame
+import math
+
+# Images
+gameover = pygame.image.load('gameover1.bmp')
+
+# Colors on rgb scale
+BLACK = (0,0,0)
+WHITE = (255,255,255)
+BLUE = (0,0,255)
+DIMGRAY = (105,105,105)
+SLATEGRAY = (112,128,144)
+
+"""Model classes"""
+class Player(object):
+	def __init__(self,x=0,y=0,width=50,height=50,dx=1,dy=.001,shiftdx=0,jumpdy=-.75):
+		# places player centered above the coordinate given
+		self.x = x
+		self.y = y-height
+		#size of player cube
+		self.width = width
+		self.height = height
+		self.dx = dx
+		self.dy = dy
+		#shiftdxs are for moving everything when the player moves
+		self.shiftdx = shiftdx
+		self.jumpdy = jumpdy # variable dy is set to when controller jumps
+
+	def train_wreck(self, train):
+		#train collision function
+		return (train.x+train.width) > self.x
+
+	def shift_world(self):
+		#shift world to left when past a certain point
+		return self.x > 350
+
+	def go_back(self):
+		#shift world to right when past a certain point
+		return self.x < 130
+
+	def hit_ground(self,ground):
+		#ground collision function for jumping
+		return (self.y + self.height) > ground.y and self.x < (ground.x+ground.width) and self.x > ground.x
+
+	def off_screen(self):
+		#if player goes off screen
+		return self.y > 480
+
+class PainTrain(object):
+	def __init__(self,x=0,y=0,width=200,height=200,constdx=.05,dx=0,shiftdx=-1):
+		# places train centered above coordinate given
+		self.x = x
+		self.y = y-height
+		#size of chasing cube
+		self.width = width
+		self.height = height
+		#constant velocity
+		self.constdx = constdx
+		self.dx = dx
+		self.shiftdx = shiftdx
+
+	def step(self):
+		self.x += self.constdx
+
+# classes for level objects
+class Ground(object):
+	def __init__(self, x = 0, y = 300, width = 2400, height = 180,dx=0,shiftdx=-1):
+		self.x = x
+		self.y = y
+		#size of ground
+		self.width = width
+		self.height = height
+		self.dx = dx
+		#shiftdx is for moving the ground when player moves
+		self.shiftdx = shiftdx
+
+class Platform(object):
```

Comment:
Ground and Platform don't seem significantly different in function and attributes. Maybe you could consolidate these into one class Platform, and then make one big platform the ground, and color it differently.